pkg/linux/hlt.go

```diff
@@ -0,0 +1,21 @@
+package linux
```

Comment:
Not a fan of that generic a package name - This whole project is already linux-specific with the usage of kexec

Reply:
Do you have a recommendation ? utils, tools ??

Reply:
I am generally wary of using names like `utils` or `tools` because it means "all the things I have no idea where to put" and in my experience this risks to end up in a messy state. What's the plan? Do you plan to add more power-related functions? If so the package name could be named after "power" or a variant, and expose functions like `power.Reset` and so on